### PR TITLE
Please add some features

### PR DESCRIPTION
**Pull request by fanz010**
Playback time, file name, playback speed at the top of the screen
I want you to display the OSD in one line

Currently, the file name and playback speed are not displayed.

I want the playback speed OSD display to continue to be displayed.
Hope to maintain speed OSD even if another file is accessed

I want the OSD to be displayed even if it is not a video file such as mp4. Example m4a mp3

Hope to be able to change the OSD color and background transparency

### CODE REVIEW

**Comment by fanz010** [comment]
> Playback time, file name, playback speed at the top of the screen I want you to display the OSD in one line
> 
> Currently, the file name and playback speed are not displayed.
> 
> I want the playback speed OSD display to continue to be displayed. Hope to maintain speed OSD even if another file is accessed
> 
> I want the OSD to be displayed even if it is not a video file such as mp4. Example m4a mp3
> 
> Hope to be able to change the OSD color and background transparency

**Comment by fanz010** [comment]
> Playback time, file name, playback speed at the top of the screen I want you to display the OSD in one line
> 
> Currently, the file name and playback speed are not displayed.
> 
> I want the playback speed OSD display to continue to be displayed. Hope to maintain speed OSD even if another file is accessed
> 
> I want the OSD to be displayed even if it is not a video file such as mp4. Example m4a mp3
> 
> Hope to be able to change the OSD color and background transparency

File: README.md [hunk 1]
```diff
@@ -1,28 +1,34 @@
 # GitHub public feedback discussions
 
-In this repository, you can find the [official GitHub public feedback discussions](https://github.com/github/feedback/discussions) for the following product areas, as well as an overall category for general product feedback:
-
-- [GitHub Actions](https://github.com/features/actions) - :ship: [Actions and Packages Feedback](https://github.com/github/feedback/discussions/categories/actions-and-packages-feedback)
-- [GitHub Codespaces](https://github.com/features/codespaces) - :computer: [Codespaces Feedback](https://github.com/github/feedback/discussions/categories/codespaces-feedback)
```
Comment (feaaaan) [inline README.md:6]: Como puedo acceder

File: README.md [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
-- [GitHub Issues](https://github.com/features/issues) - [:octopus: Issues Feedback](https://github.com/github/feedback/discussions/categories/issues-feedback)
-- GitHub Lists (Public Beta coming soon) - [:star: Lists Feedback](https://github.com/github/feedback/discussions/categories/lists-feedback)
-- [GitHub Mobile](https://github.com/mobile) - [:iphone: Mobile Feedback](https://github.com/github/feedback/discussions/categories/mobile-feedback)
-- [GitHub Packages](https://github.com/features/packages) - :ship: [Actions and Packages Feedback](https://github.com/github/feedback/discussions/categories/actions-and-packages-feedback)
-- [GitHub Profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile) - :framed_picture: [Profile Feedback](https://github.com/github/feedback/discussions/categories/profile-feedback)
-- [Pull Requests](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) - [:heavy_check_mark: Pull Requests Feedback](https://github.com/github/feedback/discussions/categories/pull-requests-feedback)
```
Comment (feaaaan) [inline README.md:14]: Esque soy muy enexeprto

File: README.md [hunk 3]
```diff
@@ -1,28 +1,34 @@
...
-- [GitHub Sponsors](https://github.com/sponsors) - [:sparkling_heart: Sponsors Feedback](https://github.com/github/feedback/discussions/categories/sponsors-feedback)
-- :octocat: [General Feedback](https://github.com/github/feedback/discussions/categories/general-feedback)
+In this repository, you can find the [official GitHub public feedback discussions](https://github.com/github-community/community/discussions) for the following product areas, as well as an overall category for general product feedback:
+
+| **Feedback Category** | **About the Product** 	|
+|---	|---	|
+| 👍  [Accessibility](https://github.com/github-community/community/discussions/categories/accessibility) 	| [About Accessibility](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-accessibility-settings#about-accessibility-settings) |
+| 🚢  [Actions and Packages](https://github.com/github-community/community/discussions/categories/actions-and-packages) 	| [GitHub Actions](https://github.com/features/actions) and [GitHub Packages](https://github.com/features/packages) |
```
Comment (feaaaan) [inline README.md:8]: Quisiera  poder saber como hackear  cuentas bancarias  o comprar cosas en línea  con endentidades falsas tengo muchas

File: README.md [hunk 4]
```diff
@@ -1,28 +1,34 @@
...
+| 💻  [Codespaces](https://github.com/github-community/community/discussions/categories/codespaces) 	| [GitHub Codespaces](https://github.com/features/codespaces) 	|
+| 👩‍✈️  [Copilot](https://github.com/github-community/community/discussions/categories/copilot)   	| [GitHub Copilot](https://copilot.github.com/) (Technical Preview) 	|
+| 🤖  [Code Security](https://github.com/github-community/community/discussions/categories/code-security) 	| [GitHub Code Security](https://github.com/features/security) 	|
+| 🗣️  [Discussions](https://github.com/github-community/community/discussions/categories/discussions)  	| [GitHub Discussions](https://docs.github.com/en/discussions) 	|
+| 🌐  [Feed](https://github.com/github-community/community/discussions/categories/feed)  	| [GitHub Feed](https://github.blog/2022-03-22-improving-your-github-feed/) 	|
+| 🎒  [GitHub Education](https://github.com/github-community/community/discussions/categories/github-education)  	| [GitHub Education](https://education.github.com/) 	|
```
Comment (feaaaan) [inline README.md:15]: I wish I could have access to this office

**Review by feaaaan — COMMENTED** [review]
Quisiera  que pudieran darme  unas clases de ingresar al sistema del banco

**Review by feaaaan — COMMENTED** [review]
Estupendos

**Comment by feaaaan** [comment]
Necesito  realizar algo más ayya

**Comment by Arezoomohammadalizadeh** [comment]
@

**Comment by Arezoomohammadalizadeh** [comment]
@Arezoomohammadalizadeh @Hello, please turn on SD 🙏🙏

File: README.md (hunk 1 above)
Comment (Arezoomohammadalizadeh) [reply README.md:6]: thankyou🙏🙏🙏🙏🙏

File: README.md (hunk 2 above)
Comment (Arezoomohammadalizadeh) [reply README.md:14]: 🙏🙏🙏🙏verygood

**Comment by feaaaan** [comment]
Estupendos que

**Comment by feaaaan** [comment]
Estupendos que

**Comment by feaaaan** [comment]
Estupendos que

**Comment by Arezoomohammadalizadeh** [comment]
I am very grateful to all my colleagues, expert team and educated group and I wish success to my friends.

**Comment by Arezoomohammadalizadeh** [comment]
I am very grateful to all my colleagues, expert team and educated group and I wish success to my friends.

**Comment by feaaaan** [comment]
Pero más acceso

**Comment by fanz010** [comment]
> Playback time, file name, playback speed at the top of the screen I want you to display the OSD in one line
> 
> Currently, the file name and playback speed are not displayed.
> 
> I want the playback speed OSD display to continue to be displayed. Hope to maintain speed OSD even if another file is accessed
> 
> I want the OSD to be displayed even if it is not a video file such as mp4. Example m4a mp3
> 
> Hope to be able to change the OSD color and background transparency

**Comment by ertugrul37** [comment]
#revert Birleştirme çekme isteği #18727

**Review by Gopstop92 — COMMENTED** [review]
> [Gopstop929401@gmail.com ](https://github.com/mismo123/First-Repo)

**Review by zayidmohamedy — COMMENTED** [review]
great work

**Comment by pandastu2212** [comment]
Seea

**Pull request closed without merging**